src/aks-preview/HISTORY.rst

```diff
@@ -15,6 +15,10 @@ Pending
 * Update the enum for `--os-sku` in command `az aks nodepool update` to only accept the expected Ubuntu and AzureLinux OSSKUs.
 * Update description `az aks update` and remove description about outbound ip limit.
 
+0.5.172
++++++++
+* Prompt when stopping a private link cluster.
```

Comment:
please also move those notes under the pending section (several lines above) to this block

Reply:
Please note that adding the default Prompt step to an existing command can cause a breaking change in the automation scripts. Please evaluate the impact of this breaking change on customers. 
If this impact is acceptable, please add prefix `[BREAKING CHANGE]` to the release note

Reply:
thanks, let me double confirm with PM.